src/rdb/rdb_raft.c

```diff
@@ -404,12 +404,13 @@ rdb_raft_pack_chunk(daos_handle_t lc, struct rdb_raft_is *is, d_iov_t *kds,
 	sgl.sg_iovs = data;
 	arg.sgl = &sgl;
 
+	arg.copy_data_cb = vos_iter_copy;
 	/* Attempt to inline all values until recx bulks are implemented. */
 	arg.inline_thres = 1 * 1024 * 1024;
 
 	/* Enumerate from the object level. */
 	rc = dss_enum_pack(&param, VOS_ITER_OBJ, true, &anchors, &arg,
-			   NULL /* dth */);
+			   vos_iterate, NULL /* dth */);
```

Comment:
I think we don't want RDB to call object API, probably that's why we put the dss_enum_pack() in iosrv at the very beginning?

Reply:
not sure the reason behind it, but we need call these object streaming api for client object.